.github/workflows/main.yml

```diff
@@ -41,6 +43,8 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+      - name: Remove /usr/local/* to free disk space
+        run:  sudo rm -rf /usr/local/*
```

Comment:
Shouldn't we apply this consistently to all build jobs?
I still wonder how to reuse steps with that damn gitlab syntax...

Reply:
I will look into it.